package.json

```diff
@@ -169,6 +169,7 @@
 		"node": "12.13.0"
 	},
 	"devDependencies": {
+		"@firebase/database-types": "^0.6.1",
```

Comment:
This is the only dependency I added here, the rest of this just appears to be alphabetization.